Riot/AppDelegate.m

```diff
@@ -2143,7 +2144,7 @@ - (void)parseUniversalLinkFragment:(NSString*)fragment outPathParams:(NSArray<NS
     NSMutableArray<NSString*> *pathParams2 = [NSMutableArray arrayWithArray:pathParams];
     for (NSInteger i = 0; i < pathParams.count; i++)
     {
-        pathParams2[i] = [pathParams2[i] stringByReplacingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
+        pathParams2[i] = [pathParams2[i] stringByRemovingPercentEncoding];
```

Comment:
Argh. We need the sibling method `[MXTools decodeURIComponent:]` to have perfect inverse.

Reply:
What would it do differently from stringByRemovingPercentEncoding?

Reply:
You are right.
I thought (without checking iOS API) that a method like `stringByRemovingPercentEncodingWithAllowedCharacters:` existed but I was wrong.